internal/action/commands.go

```diff
@@ -815,17 +814,17 @@ func (s *Action) GetCommands() []*cli.Command {
 				},
 				&cli.StringFlag{
 					Name:  "crypto",
-					Usage: "Select crypto backend (gpg, gpgcli, plain, xc)",
+					Usage: fmt.Sprintf("Select crypto backend %v", backend.CryptoBackends()),
 				},
 				&cli.StringFlag{
 					Name:  "rcs",
```

Comment:
Why `rcs` here and `storage` for the other ones?

Reply:
Actually we'll have to go over all the command flags. I guess more than one will be inconsistent / ineffective.
I plan on doing that later.